src/pages/methods/workflow-details/WorkflowWrapper.ts

```diff
@@ -289,6 +293,31 @@ export const WorkflowsContainer = (props: WorkflowContainerProps) => {
         setPermissionsModalOpen,
         refresh: loadSnapshot,
       }),
+    showCloneModal &&
+      h(WorkflowModal, {
+        title: 'Clone snapshot',
+        defaultName: name.concat('_copy'),
+        defaultWdl: snapshot?.payload,
+        defaultDocumentation: snapshot?.documentation,
+        defaultSynopsis: snapshot?.synopsis,
+        defaultSnapshotComment: snapshot?.snapshotComment,
+        buttonActionName: 'Clone snapshot',
```

Comment:
I am a little hesitant to call it `Clone Method` or `Clone Workflow` because this functionality is only cloning one snapshot of the method/workflow. I think that calling it clone method/workflow might indicate that it clones all the snapshots in the method, which it doesn't do.

Maybe we do something like this?
- call the action menu "Clone snapshot"
- call the modal title "Create new Method"
- call modal success button "Create new Method"

This might indicate it is creating a new method of that snapshot? But the naming across all this will be inconsistent.

Or we can add a tooltip or some description in the modal that it will indicate this will create a new method using that snapshot?

Reply:
> Maybe we do something like this?
> 
> - call the action menu "Clone snapshot"
> - call the modal title "Create new Method"
> - call modal success button "Create new Method"

I like this idea, because really all that the "clone" is doing is allowing us to create a new workflow with the current snapshot used as a template. But I agree that the difference in naming from the button to the modal might be confusing. Maybe we should check with Beth?

Reply:
Yeah I can check with Beth 👍